controllers/configmap.go

```diff
@@ -83,85 +88,130 @@ const accessLogFormat = `[%START_TIME%] %BYTES_RECEIVED% %BYTES_SENT% %DURATION%
 `
 
 func envoyConfig(es *egressv1.ExternalService) (string, error) {
-	config := bootstrapv2.Bootstrap{
-		Node: &envoycorev2.Node{
+	config := bootstrap.Bootstrap{
+		Node: &envoycorev3.Node{
 			Cluster: es.Name,
 		},
-		Admin: &bootstrapv2.Admin{
-			Address: &envoycorev2.Address{Address: &envoycorev2.Address_SocketAddress{
-				SocketAddress: &envoycorev2.SocketAddress{
+		Admin: &bootstrap.Admin{
+			Address: &envoycorev3.Address{Address: &envoycorev3.Address_SocketAddress{
+				SocketAddress: &envoycorev3.SocketAddress{
 					Address:  "0.0.0.0",
-					Protocol: envoycorev2.SocketAddress_TCP,
-					PortSpecifier: &envoycorev2.SocketAddress_PortValue{
+					Protocol: envoycorev3.SocketAddress_TCP,
+					PortSpecifier: &envoycorev3.SocketAddress_PortValue{
 						PortValue: uint32(adminPort(es)),
 					},
 				},
 			}},
 			AccessLogPath: "/dev/stdout",
 		},
-		StaticResources: &bootstrapv2.Bootstrap_StaticResources{},
+		StaticResources: &bootstrap.Bootstrap_StaticResources{},
 	}
 
 	for _, port := range es.Spec.Ports {
 		protocol := protocolToEnvoy(port.Protocol)
 		name := fmt.Sprintf("%s_%s_%s", es.Name, envoycorev2.SocketAddress_Protocol_name[int32(protocol)], strconv.Itoa(int(port.Port)))
```

Comment:
```suggestion
		name := fmt.Sprintf("%s_%s_%s", es.Name, envoycorev3.SocketAddress_Protocol_name[int32(protocol)], strconv.Itoa(int(port.Port)))
```

I think it'd make more sense to use corev3 here (assuming a similar mapping exists!)